asm/rom_8011DC0.s

```diff
@@ -745,7 +745,7 @@ _08017228:
 	bl DestroyListMenuTask
 	ldrb r0, [r5]
 	movs r1, 0x1
-	bl sub_819746C
+	bl ClearWindowAndBorder
```

Comment:
There are many functions that do this same functionality, you should probably distinguish them more clearly